src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -221,11 +231,358 @@ function svdTTN(
   return H
 end
 
+function qn_svdTTN(
+  os::OpSum, sites::IndsNetwork, root_vertex::VT; kwargs...
+)::TTN where {VT}
+  # Function barrier to improve type stability
+  coefficient_type = ITensors.determineValType(terms(os))
+  return qn_svdTTN(coefficient_type, os, sites, root_vertex; kwargs...)
+end
+
+function qn_svdTTN(
+  coefficient_type::Type{<:Number},
+  os::OpSum,
+  sites::IndsNetwork,
+  root_vertex::VT;
+  mindim::Int=1,
+  maxdim::Int=typemax(Int),
+  cutoff::Float64=eps(coefficient_type) * 1e1,
```

Comment:
I would remove `::Float64` since `eps` is meant to handle cases like single precision.

Also I think `* 10` is easier to read than `1e1` and will help preserve element types better.

Finally, I think we want `eps(real(coefficient_type))` in case `coefficient_type` is complex, since `cutoff` should be real.